service/application/src/test/java/com/solace/maas/ep/event/management/agent/scanManager/rest/DataImportControllerTests.java

```diff
@@ -56,8 +56,6 @@ public void testDataImportControllerRead() {
 
         assertThat(reply.getStatusCode()).isEqualTo(HttpStatus.OK);
         assertThat(reply.getBody()).contains("Manual import started.");
-
-        assertThatNoException();
```

Comment:
Not for these tests. A few things to note here:
- assertThatNoException() isn't used this way so it was being useless here.
- If we want to test that there's no exception, we should just check the status code because this code is set up to return a 400 on error as seen in other tests as well
- Any other uncaught exceptions will fail the test